packages/rancher-windows-exporter/charts/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if and .Values.serviceMonitor .Values.clients }}{{- if and .Values.serviceMonitor.enabled .Values.clients.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels: {{ include "windowsExporter.labels" . | nindent 4 }}
+  name: {{ template "windowsExporter.name" . }}
+  namespace: {{ template "windowsExporter.namespace" . }}
+spec:
+  selector:
+    matchLabels: {{ include "windowsExporter.labels" . | nindent 6 }}
+  namespaceSelector:
+    matchNames:
+    - {{ template "windowsExporter.namespace" . }}
+  endpoints:
+  - port: windows-metrics
+    metricRelabelings:
+    - sourceLabels: [__name__]
+      regex: 'wmi_(.*)'
+      replacement: 'windows_$1'
+      targetLabel: __name__
+    - sourceLabels: [__name__]
+      regex: windows_mssql_transactions_active_total
+      replacement: 'windows_mssql_transactions_active'
+      targetLabel: __name__
```

Comment:
FYI this comes from https://github.com/prometheus-community/windows_exporter/releases/tag/v0.15.0